idseq_dag/steps/run_lzw_deprecated.py

```diff
@@ -0,0 +1,130 @@
+from idseq_dag.engine.pipeline_step import PipelineStep
+import idseq_dag.util.command as command
+import idseq_dag.util.log as log
+import idseq_dag.util.count as count
+
+class PipelineStepRunLZW__DEPRECATED(PipelineStep):
```

Comment:
tbh we can probably just delete this already?

Reply:
thanks for the suggestion, i will delete it after some testing